fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -766,6 +766,16 @@ public class Config extends ConfigBase {
     @ConfField
     public static int publish_version_interval_ms = 10;
 
+    @ConfField(mutable = true)
+    public static boolean enable_lake_batch_publish_version = false;
```

Comment:
```suggestion
    public static boolean lake_enable_batch_publish_version = false;
```
For consistent naming style with others.